x-pack/plugins/observability_solution/infra/server/lib/alerting/common/utils.ts

```diff
@@ -173,8 +173,10 @@ export const getMetricsViewInAppUrlWithSpaceId = ({
   groupBy?: string[];
   assetDetailsLocator?: LocatorPublic<AssetDetailsLocatorParams>;
   metricsExplorerLocator?: LocatorPublic<MetricsExplorerLocatorParams>;
+  additionalContext?: AdditionalContext;
 }) => {
   const fields = {
+    ...hasAdditionalContext,
```

Comment:
Shouldn't here be `additionalContext`?

Reply:
That must have been an autocomplete failure or a brainfart on my part. Also, lol, can spread a function into an object and have that be seen as "fine" by TS.